sound/soc/sof/ipc4-topology.c

```diff
@@ -146,6 +147,8 @@ static int get_token_process_type(void *elem, void *object, u32 offset)
 static const struct sof_topology_token process_tokens[] = {
 	{SOF_TKN_EFFECT_TYPE, SND_SOC_TPLG_TUPLE_TYPE_STRING, get_token_process_type,
 		offsetof(struct sof_ipc4_process, process_type)},
+	{SOF_TKN_EFFECT_CPC_LP, SND_SOC_TPLG_TUPLE_TYPE_WORD, get_token_u32,
+		offsetof(struct sof_ipc4_process, cpc_low_power_mode)},
```

Comment:
this needs to be squahsed with the previous patch. And what is CPC lower power mode?

Reply:
OK, I will do it. I'm not sure what CPC lower power mode is. This is required by cAVS FW. I guess in normal mode, it will cpc. And in lower power mode (ULP?), it will use the cpc_low_power_mode.

Reply:
what is CPC?

Reply:
> what is CPC?

from comments of cAVS: cycles per single run